### PR TITLE
Corrections de français + coquille

### PR DESCRIPTION
**Pull request by Pontax**

### CODE REVIEW

File: fr/development/routing.rst [hunk 1]
```diff
@@ -830,7 +830,7 @@ utilisés lors de la connexion des routes. Par exemple, si votre action
 renommer vos actions::
 
     $routes->resources('Articles', [
-        'actions' => ['update' => 'put', 'create' => 'create']
+        'actions' => ['update' => 'put', 'add' => 'create']
```
Comment (markstory) [inline fr/development/routing.rst]: The key is actually `create`.

Comment (Pontax) [inline fr/development/routing.rst]: Both key and value are "create" ?

Comment (HavokInspiration) [inline fr/development/routing.rst]: They could be.
The key is `create` and the value can be anything.
@markstory For the sake of the example, maybe the value should be different ?

Comment (HavokInspiration) [inline fr/development/routing.rst]: Well reading the line below this one, it should actually be `'actions' => ['update' => 'put', 'create' => 'add']`

Comment (cake17) [inline fr/development/routing.rst]: @Pontax are you ok with @HavokInspiration proposal? If yes, could you also make the change for english version ?

Comment (Pontax) [inline fr/development/routing.rst]: Sorry for the (very) late answer. 
Of course, I'm OK.

**Pull request merged**